zeeguu_api/api/search.py

```diff
@@ -0,0 +1,172 @@
+import flask
+import zeeguu
+from zeeguu.model.search import Search
+from zeeguu.model.search_filter import SearchFilter
+from zeeguu.model.search_subscription import SearchSubscription
+from zeeguu.content_recommender.mixed_recommender import article_search_for_user
+
+from .utils.route_wrappers import cross_domain, with_session
+from .utils.json_result import json_result
+from . import api
+
+session = zeeguu.db.session
+
+SEARCH = "search"
+SUBSCRIBE_SEARCH = "subscribe_search"
+UNSUBSCRIBE_SEARCH = "unsubscribe_search"
+SUBSCRIBED_SEARCHES = "subscribed_searches"
+FILTER_SEARCH = "filter_search"
+UNFILTER_SEARCH = "unfilter_search"
+FILTERED_SEARCHES = "filtered_searches"
+
+
+# ---------------------------------------------------------------------------
+@api.route(f"/{SUBSCRIBE_SEARCH}/<search_terms>", methods=("GET",))
+# ---------------------------------------------------------------------------
+@cross_domain
+@with_session
+def subscribe_to_search(search_terms):
+    """
+    :param: search_terms -- the search terms to be subscribed to.
+    Subscribe to a search with given search terms.
+
```

Comment:
add return comment. 
maybe even example

Reply:
"the search is the thing to display / subscription is used by the backend to process incoming articles"